src/dense.jl

```diff
@@ -48,12 +48,28 @@ end
 
 Base.length(dense::DenseOutput) = length(dense.opts.tout)
 
+tdir(ivp, solver::DenseOutput) = sign(solver.opts.tstop - ivp.t0)
+
 @compat function (::Type{DenseOutput{I}}){T,I}(ivp::IVP{T};
+                                               tstop = T(Inf),
+                                               tout::AbstractVector{T} = T[tstop],
```

Comment:
Maybe we should make `tout` a non-optional argument. What is the point of calling a dense solver to only output `tstop`?